journal/updated/02-chapter-1.adoc

```diff
@@ -162,33 +162,36 @@ Leicester, 27th of Sixth month, 1754.
 [.salutation]
 Dear Friend,
 
-I with gladness received your letter, but in the reading of it,
+I with gladness received your letter, but in the reading of it
 was seized with a concern that you are not yet emptied enough of self.
-I hope, my dear friend, you will bear with me, as I must deal honestly;
+I hope, my dear friend, you will bear with me, as I must deal honestly,
 and desire I may speak simply upon all occasions without
 seeking to ornament it with what might please myself,
-because I have learned to know that what is done to gratify self,
+because I have learned to know that what is done to gratify self
 is displeasing to the Almighty.
 It seems to me as if you were in a track that I long trod in,
-therefore I am in much love engaged to tell you a little of what I have experienced.
-When the Lord has been pleased to lay a concern upon me to speak among his people,
+and therefore I am in much love engaged to tell you a little of what I have experienced.
+When the Lord has been pleased to lay a concern upon me to speak among His people,
 I was not willing to stand up or to kneel down, until I had as I thought,
```

Comment:
maybe this would be easier to understand if we change it to: until I had (as I thought)